app/views/deposits/fixed/newapplication.html

```diff
@@ -67,7 +69,8 @@
                             <select id="interestCompoundingPeriodType"
                                     ng-model="formData.interestCompoundingPeriodType"
                                     ng-options="type.id as type.value for type in data.interestCompoundingPeriodTypeOptions"
-                                    value="{{type.id}}" class="form-control width170px"></select>
+                                    value="{{type.id}}" class="form-control width170px" name="interestCompoundingPeriodType"></select>
+                            <form-validate valattributeform="newsavingccountform" valattribute="interestCompoundingPeriodType"/>
```

Comment:
No need for `form-validate` here since this is a default value, there is no way the user can miss out this field.